src/Microsoft.Kusto.ServiceLayer/DataSource/Kusto/KustoDataSource.cs

```diff
@@ -494,32 +560,74 @@ private void LoadTableSchema(DataSourceObjectMetadata databaseMetadata)
             {
                 return;
             }
-            
+
             var rootTableFolderKey = new StringBuilder($"{databaseMetadata.Urn}");
             if (tableInfos.Any(x => !string.IsNullOrWhiteSpace(x.Folder)))
             {
-                // create Table folder to hold functions tables
+                // create Table folder to hold tables folders
                 var tableFolder = MetadataFactory.CreateFolderMetadata(databaseMetadata, rootTableFolderKey.ToString(), "Tables");
-                _folderMetadata.AddRange(rootTableFolderKey.ToString(), new List<FolderMetadata> {tableFolder});
+                _folderMetadata.AddRange(rootTableFolderKey.ToString(), new List<FolderMetadata> { tableFolder });
                 rootTableFolderKey.Append($".{tableFolder.Name}");
-                
+
                 SetFolderMetadataForTables(databaseMetadata, tableInfos, rootTableFolderKey.ToString());
             }
-            
+
             SetTableMetadata(databaseMetadata, tableInfos, rootTableFolderKey.ToString());
         }
 
+        private IEnumerable<TableInfo> GetMaterializedViewsInfos(string databaseName)
+        {
+            CancellationTokenSource source = new CancellationTokenSource();
```

Comment:
In other variables you start with var, couldn't these two be started with var too?

`var source = new CancellationTokenSource();`
`var token = source.Token;`